README.md

```diff
@@ -7,6 +7,16 @@ This is the developer documentation. For user documentation, please consult http
 
 ## Building
 
+Make sure you have all dependencies installed for the Aerospike C client and asbackup.
+See https://github.com/aerospike/aerospike-client-c#build-prerequisites for more C client information.
+Below are dependencies for asbackup only.
+- openssl (1.1 or 3)
+- An event library: libuv, libevent, or libev (also used by the C client submodule)
```

Comment:
This one is optional

Reply:
The build errors out if no event lib is used. #error "Must define an event library when building"